### DISCUSSION TITLE
stylus code snippets aren't supported very well with volar

### DISCUSSION THREAD

**Discussion by zouhangwithsweet** [vuejs/language-tools#265] [General]
I have to remember all css properties

**Comment by johnsoncodehk** [comment]
Please wait for Plugiin API: https://github.com/johnsoncodehk/volar/issues/185

But I strongly recommend that you switch to css / less / scss. see https://github.com/johnsoncodehk/volar#note.

**Reply by zouhangwithsweet** [reply]
Thanks for your reply.

**Comment by johnsoncodehk** [comment]
Sorry I may misunderstood your question, stylus snippets is supported. You need to use `<style lang="stylus">` and install stylus code snippets support extension you want.

https://marketplace.visualstudio.com/search?term=stylus&target=VSCode&category=All%20categories&sortBy=Relevance
